openwisp_users/models.py

```diff
@@ -25,6 +26,24 @@ class Organization(BaseOrganization, AbstractOrganization):
     class Meta(AbstractOrganization.Meta):
         swapper.swappable_setting('openwisp_users', 'Organization')
 
+    def add_user(self, user, **kwargs):
+        """
+        overriding this method of ``AbstractOrganization``
+        to remove organization owner creation logic as it is
+        being handled separately by a signal of this project.
+        """
+
+        users_count = self.users.all().count()
```

Comment:
since we are at it, we can use `exists()` which is more efficient, will return `True` or `False`.